### ISSUE TITLE
`log` fails to build for thumbv4t-none-eabi

### ISSUE THREAD

**Issue by wtetzner** [rust-lang/log#487]
```
    Blocking waiting for file lock on package cache
    Updating crates.io index
    Blocking waiting for file lock on package cache
   Compiling log v0.4.14
   Compiling cfg-if v1.0.0
error[E0599]: no method named `compare_exchange` found for struct `AtomicUsize` in the current scope
    --> /Users/walter/.cargo/registry/src/github.com-1ecc6299db9ec823/log-0.4.14/src/lib.rs:1347:33
     |
1347 |     let old_state = match STATE.compare_exchange(
     |                                 ^^^^^^^^^^^^^^^^ method not found in `AtomicUsize`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `log` due to previous error
```

**Comment by KodrAus** [comment]
Hi @wtetzner :wave: We've got a build script that flags targets that don't have atomics support. It looks like we're just missing the `thumbv4t-none-eabi` target here: https://github.com/rust-lang/log/blob/master/build.rs#L36

**Comment by KodrAus** [comment]
This is included in `0.4.15`. Thanks again @wtetzner!
